data/apps.yml

```diff
@@ -58,15 +58,15 @@
     "128":  "https://flathub.org/repo/appstream/x86_64/icons/128x128/org.gimp.GIMP.png"
     "64":   "https://flathub.org/repo/appstream/x86_64/icons/64x64/org.gimp.GIMP.png"
     
-- name:     "Atom"
-  id:       "io.atom.Atom"
-  ref:      "app/io.atom.Atom/x86_64/stable"
-  summary:  "A hackable text editor for the 21st Century"
-  color:    "#96CBA5"
-  "flatpakref": "https://flathub.org/repo/appstream/io.atom.Atom.flatpakref"
+- name:     "Firefox"
+  id:       "org.mozilla.firefox"
+  ref:      "app/org.mozilla.firefox/x86_64/stable"
+  summary:  "Fast, Private & Safe Web Browser"
+  color:    "#FFDA42"
```

Comment:
@jimmac Is this a good color for the Firefox icon or should I select a different one?

Reply:
Just checked, if they set something in their appdata, unfortunatly not.

https://hg.mozilla.org/mozilla-central/file/tip/taskcluster/docker/firefox-flatpak/org.mozilla.firefox.appdata.xml.in

Reply:
Well, I think it is not used. :-) I vaguely remember that there used to be a page with featured Flatpak apps on Flatpak.org many, many years ago. This is probably what remained from it. But I would still like to set the color to a correct looking one. :-) I will probably go with `#FFEF42`.